service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/common/ExtendedAuthorizationRequest.java

```diff
@@ -60,13 +63,16 @@
 
     private List<Integer> va;
 
-    protected BaseKlarnaRequest(final PayoneConfig config, final String requestType,
-                                final String financingtype, final PaymentWithCartLike paymentWithCartLike) {
-        super(config, requestType, ClearingType.PAYONE_KLV.getPayoneCode());
-
-        this.financingtype = financingtype;
-
-        mapKlarnaPropertiesFromPaymentCart(paymentWithCartLike);
+    protected ExtendedAuthorizationRequest(final PayoneConfig config,
+                                           final String requestType,
+                                           final String financingtype,
+                                           final String clearingType,
+                                           final PaymentWithCartLike paymentWithCartLike) {
+        super(config, requestType, clearingType);
+        if (StringUtils.isNotBlank(financingtype)) {
```

Comment:
I think there is no need to check if the string is blank (`""`) here

Reply:
3367ea973